components/p3a/features.cc

```diff
@@ -0,0 +1,14 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/p3a/features.h"
+
+namespace brave {
+namespace features {
+
+BASE_FEATURE(kSTAR, "BraveP3ASTAR", base::FEATURE_DISABLED_BY_DEFAULT);
```

Comment:
pls add a comment / link to spec or ticket

Reply:
nit: is there a reason the feature symbol does not match the name?